.travis.yml

```diff
@@ -3,5 +3,6 @@ language: node_js
 node_js:
   - "4"
   - "6"
+  - "8.7.0"
```

Comment:
Why this version exactly?

Reply:
:trollface:  oh man, yea. took me a bit to peek through their changelog to sort that out. 🎺